conda/recipes/raft-ann-bench/meta.yaml

```
@@ -78,6 +78,8 @@ requirements:
     - h5py {{ h5py_version }}
     - benchmark
     - matplotlib
+    # rmm is needed to determine if package is gpu-enabled
+    - rmm = {{ minor_version }}
```

```suggestion
    - rmm ={{ minor_version }}
```

Minor nit for consistency with the rest of this file & other RAPIDS recipes.